messaging/kafka-streams-reactive-messaging/src/main/resources/application.properties

```diff
@@ -0,0 +1,42 @@
+# Configuration file
+quarkus.http.port=8080
+quarkus.http.test-port=8080
+
+producer.httpCodes=200,201,400,401,403,500
+producer.loginUrls=redhat/login,inditex/login,santander/login,bbva/login
+sse.endpoint=http://localhost:8080/monitor/stream
```

Comment:
As far I could see, we don't need this property. We only need the port for local testing (which is already mapped using another property). For integration testing, this is resolved using the `@TestResource` annotation.

Reply:
I think this is a leftover now.